ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py

```diff
@@ -17,6 +17,34 @@
 )
 
 
+#cond
+@handle_frontend_test(
+    fn_tree="jax.numpy.linalg.cond",
+    dtype_and_x=_get_dtype_and_matrix_non_singular(),
```

Comment:
`_get_dtype_and_matrix_non_singular()`, is not defined and implemented anywhere.